cypress/e2e/wip/Crypto/crypto_deposit_check.cy.js

```diff
@@ -1,83 +1,83 @@
-describe('QATEST-715 - Load crypto deposit page', () => {
-  beforeEach(() => {
-    cy.c_login({ user: 'allcrypto' })
-    cy.c_visitResponsive('/appstore/traders-hub', { size: 'desktop' })
-  })
-
-  it('should be able to create crypto account from Traders Hub.', () => {
-    const cryptocurrencies = [
-      'Bitcoin',
-      'Ethereum',
-      'Litecoin',
-      'Tether ERC20',
-      'USD Coin',
-      'Tether TRC20',
-    ]
-    cryptocurrencies.forEach((crypto) => {
-      cy.findByText('Options & Multipliers').should('be.visible')
-      cy.c_closeNotificationHeader()
-      cy.findByTestId('dt_currency-switcher__arrow')
-        .should('be.visible')
-        .click()
-      cy.findByText(crypto).click()
-      cy.findByTestId('dt_currency-switcher__arrow').should('be.visible')
-      cy.c_closeNotificationHeader()
-      cy.findByRole('button', { name: 'Deposit' }).click()
+const sizes = ['mobile', 'desktop']
+sizes.forEach((size) => {
+  describe(`QATEST-00000 - Test Case summary here on ${size}`, () => {
+    beforeEach(() => {
+      cy.c_login({ user: 'allcrypto' })
+    })
+    it(`should be able to check crypto account from Traders Hub on ${size}`, () => {
+      cy.c_visitResponsive('', { size: size })
+      const isMobile = size == 'mobile' ? true : false
+      cy.c_visitResponsive('/appstore/traders-hub', { size: size })
```

Comment:
You want need to visit the tradershub page as cy.c_login lands on tradershub
```suggestion
      const isMobile = size == 'mobile' ? true : false
```

Reply:
Have made this change requested.